docs/dsl_specification.md

```diff
@@ -0,0 +1,150 @@
+# DSL Specification
+
+
+## How to define comments
+
+use //
+
+## How to define task
+
+* A task starts with a keyword "task" follow by the task name
```

Comment:
```suggestion
* A task starts with a keyword "task" followed by the task name
```

Reply:
I think this is a quite quick draft we need to rewrite when the parser is done.